python/tvm/relay/op/_tensor.py

```diff
@@ -1,49 +1,294 @@
 
-def add_schedule(outputs, target):
-    assert len(outputs) == 1
-    return tvm.create_schedule(outputs[0].op)
-
-register("add", "FTVMCompute", add_compute)
-register("add", "FTVMSchedule", add_schedule)
+register_compute("add", add_compute)
+register_schedule("add", schedule_injective)
 
+# subtract
 def subtract_compute(attrs, inputs, output_type, target):
     assert len(inputs) == 2
     return [topi.subtract(inputs[0], inputs[1])]
 
-def subtract_schedule(outputs, target):
-    assert len(outputs) == 1
-    return tvm.create_schedule(outputs[0].op)
-
-register("subtract", "FTVMCompute", subtract_compute)
-register("subtract", "FTVMSchedule", subtract_schedule)
+register_compute("subtract", subtract_compute)
+register_schedule("subtract", schedule_broadcast)
 
+# multiply
 def multiply_compute(attrs, inputs, output_type, target):
     assert len(inputs) == 2
     return [topi.multiply(inputs[0], inputs[1])]
 
-def multiply_schedule(outputs, target):
-    assert len(outputs) == 1
-    return tvm.create_schedule(outputs[0].op)
+register_compute("multiply", multiply_compute)
+register_schedule("multiply", schedule_broadcast)
+
+# divide
+def divide_compute(attrs, inputs, output_type, target):
+    assert len(inputs) == 2
+    return [topi.divide(inputs[0], inputs[1])]
+
+register_compute("divide", divide_compute)
+register_schedule("divide", schedule_broadcast)
+
+# pow
+def pow_compute(attrs, inputs, output_type, target):
+    assert len(inputs) == 2
+    return [topi.power(inputs[0], inputs[1])]
+
+register_compute("pow", pow_compute)
+register_schedule("pow", schedule_injective)
+
+# mod
+def mod_compute(attrs, inputs, output_type, target):
+    assert len(inputs) == 2
+    return [topi.mod(inputs[0], inputs[1])]
 
-register("multiply", "FTVMCompute", multiply_compute)
-register("multiply", "FTVMSchedule", multiply_schedule)
+register_compute("mod", mod_compute)
+register_schedule("mod", schedule_broadcast)
 
+# equal
 def equal_compute(attrs, inputs, output_type, target):
     assert len(inputs) == 2
     return [topi.equal(inputs[0], inputs[1])]
 
-def equal_schedule(outputs, target):
-    assert len(outputs) == 1
-    return tvm.create_schedule(outputs[0].op)
+register_compute("equal", equal_compute)
+register_schedule("equal", schedule_broadcast)
+
+# not_equal
+def not_equal_compute(attrs, inputs, output_type, target):
+    assert len(inputs) == 2
+    return [topi.not_equal(inputs[0], inputs[1])]
+
+register_compute("not_equal", not_equal_compute)
+register_schedule("not_equal", schedule_broadcast)
+
+# less
+def less_compute(attrs, inputs, output_type, target):
+    assert len(inputs) == 2
+    return [topi.less(inputs[0], inputs[1])]
+
+register_compute("less", less_compute)
+register_schedule("less", schedule_broadcast)
+
+# less equal
+def less_equal_compute(attrs, inputs, output_type, target):
+    assert len(inputs) == 2
+    return [topi.less_equal(inputs[0], inputs[1])]
+
+register_compute("less_equal", less_equal_compute)
```

Comment:
As per nnvm tradition, we should move most of the compute function to c++, and only leave a few(e.g. conv2d) in python

Reply:
Our current plan is to do this in a second pass.